docs/solution-pack-guide.md

```diff
@@ -0,0 +1,32 @@
+# Usage of Data Leakage Threat Response Solution Pack
+
+## Use Case Workflow
```

Comment:
I believe we don't need this file and its content. Please check with Sujit

Reply:
Discussed with Sujit, removed the files